### PR TITLE
avoid reusing fixtures

### PR DESCRIPTION
**Pull request by elharo**
@mthmulders

### CODE REVIEW

File: src/test/java/org/apache/maven/shared/utils/xml/XmlWriterUtilTest.java [hunk 1]
```diff
@@ -86,7 +75,6 @@ public void testWriteLineBreakXMLWriterInt()
     {
         XmlWriterUtil.writeLineBreak( xmlWriter, 10 );
         writer.close();
-        assertTrue( StringUtils.countMatches( output.toString(), XmlWriterUtil.LS ) == 10 );
```
Comment (mthmulders) [inline src/test/java/org/apache/maven/shared/utils/xml/XmlWriterUtilTest.java:89]: Did you remove the assertion on purpose? If so, why?

Comment (elharo) [reply src/test/java/org/apache/maven/shared/utils/xml/XmlWriterUtilTest.java:89]: nope, just an error. I'll fix it.

Comment (elharo) [reply src/test/java/org/apache/maven/shared/utils/xml/XmlWriterUtilTest.java:89]: done

**Review by mthmulders — APPROVED** [review]
Looks good to me.

**Pull request merged**